spa/src/pages/mall/MallUploadPage.vue

```diff
@@ -114,13 +115,15 @@
     </p>
 
     <div align=center>
-      <input type="button"
-             value="Accept & Upload"
-             @click="upload"
-             class="btn" />
-      <input type="button"
-             value="Back"
-             class="btn" />
+      <div class="text-center font-bold text-green" v-if="showSuccess">
+      Congratulations! Your object has been uploaded and is awaiting approval.
+    </div>
+    <input type="button"
+      value="Accept & Upload"
+      @click="upload"
+      class="btn" v-if="!showSuccess"/>
+      <button class="btn" @click="reload()" v-else>Upload More</button>
+      <router-link to="/place/mall"><button class="btn">Back</button></router-link>
```

Comment:
Just an fyi, you can add the class="btn" to the router link and remove the button.

Reply:
Fixed. :)